core-test/src/test/scala/org/bitcoins/core/psbt/PSBTTest.scala

```diff
@@ -136,6 +136,48 @@ class PSBTTest extends BitcoinSAsyncTest {
     assert(psbt2.combinePSBT(psbt2) != expected)
   }
 
+  it must "correctly combine PSBTs" in {
+    forAllAsync(PSBTGenerators.psbtWithUnknowns) { psbtF =>
```

Comment:
why with Unknowns instead of using the arbitrary generator?

Reply:
I think using the arbitrary generator will result in a more general test here

Reply:
Since the arbitrary just prunes parts of `psbtWithUnknowns` figured it'd be pointless to prune twice